platforms/BUILD

```diff
@@ -5,5 +5,4 @@ platform(
         "@platforms//cpu:armv6-m",
         "@koro-platforms//cxx:14",
     ],
-    target_platform = True,
```

Comment:
I'm using latest Bazel 7.0.0 and this option wasn't recognized; seemed safe to remove?

Reply:
Its been a while since I worked on this and it looks like things have changed in the meantime. I'm currently using bazel 5.4.1. I'll try to figure out if this is needed.